dietpi/dietpi-software

```diff
@@ -12376,6 +12406,56 @@ _EOF_
 
 		fi
 
+    software_id=181 # PaperMC
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+      
+      Create_User -d /opt/papermc papermc # This creates a system user named papermc with /opt/papermc as its home dir.
+      # Bedrock compatibility
+      G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+      if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate plugins for compatibility with Bedrock Edition?
+      \n Note that this may be buggy."; then
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar' /opt/papermc/plugins/Geyser-Spigot.jar
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
+      fi
+      G_EXEC cd /opt/papermc
+      G_EXEC_OUTPUT=1 G_EXEC java -jar /opt/papermc/paperclip.jar nogui
+      G_CONFIG_INJECT 'eula=' 'eula=true' /opt/papermc/eula.txt
+      G_EXEC chown -R papermc /opt/papermc
+      G_EXEC usermod -a -G papermc root
+      G_EXEC chmod -R 760 /opt/papermc
+      Download_Install https://github.com/Tiiffi/mcrcon/archive/v0.7.1.zip /opt/papermc/ G_EXEC unzip /opt/papermc/mcrcon.zip G_EXEC rm /opt/papermc/mcrcon.zip
```

Comment:
```
G_EXEC unzip /opt/papermc/mcrcon.zip G_EXEC rm /opt/papermc/mcrcon.zip
```
These four arguments have no effect and are not required. 
The `Download_Install` function only requires the URL and optionally the path to extract the archive to. unzip and archive removal are done automatically by the function 😎.

If finally only the `mcrcon` binary is required, I suggest to do:
```
Download_Install https://github.com/Tiiffi/mcrcon/archive/v0.7.1.tar.gz # gzip archive, better compressed + faster decompression, without path, so it extracts to the current tmpfs directory: /tmp/DietPi-Software
gcc mcrcon-0.7.1/mcrcon.c -o /opt/papermc/mcrcon # Keep in papermc directory?
rm -R mcrcon-0.7.1
strip /opt/papermc/mcrcon # This reduces the file size radically
```